client/src/index.js

```diff
@@ -7,24 +7,15 @@ socket.addEventListener('open', function (event) {
 });
 
 socket.addEventListener('message', function (event) {
-  var questiondiv = document.createElement ('div');
-  var answer1div = document.createElement ('div');
-  var answer2div = document.createElement ('div');
-  var answer3div = document.createElement ('div');
-  var answer4div = document.createElement ('div');
-  
   var obj = JSON.parse(event.data);
-  questiondiv.innerHTML = obj.question;
-  answer1div.innerHTML = obj.answers[0].answer;
-  answer2div.innerHTML = obj.answers[1].answer;
-  answer3div.innerHTML = obj.answers[2].answer;
-  answer4div.innerHTML = obj.answers[3].answer;
+  appendTextDiv(document.getElementById('main'),obj.question);
```

Comment:
When you call document.getElementById('main') you are asking the JS interpreter to go and find an element in the DOM tree with id='main'. You don't want to do this multiple times so,

`
let mainDiv = document.getElementById('main');
appendTextDiv(mainDiv, obj.question);
`